.rat-excludes

```diff
@@ -71,6 +71,7 @@ ibm-db2.svg
 postgresql.svg
 snowflake.svg
 ydb.svg
+loading.svg
```

Comment:
We CAN add the ASF boiler plate (the HTML comment flavor) into the SVG file, rather than excluding it.

Reply:
I really din't want to serve a large comment as it adds payload ...